checks/check_extra94

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2018) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+CHECK_ID_extra94="9.4"
+CHECK_TITLE_extra94="[extra94] Check if lambda functions has policies which allow access to everyone having an aws account (Not Scored) (Not part of CIS benchmark) (Custom Check)"
+CHECK_SCORED_extra94="NOT_SCORED"
+CHECK_TYPE_extra94="EXTRA"
+CHECK_SEVERITY_extra94="Critical"
+CHECK_ASFF_RESOURCE_TYPE_extra94="AwsCloudWatch"
+CHECK_ALTERNATE_check94="extra94"
+CHECK_SERVICENAME_extra94="CloudWatch"
```

Comment:
Same here, I don't think this belongs to CloudWatch

Reply:
Yes, It's typo, I will update this